### ISSUE TITLE
I want to convert this map to an accessible knowledge map

### ISSUE THREAD

**Issue by avidrucker** [knowledge-for-good/knowledgeforgood#280]
What I want to convert: https://www.reddit.com/r/computerscience/comments/khbiwd/a_map_of_computer_science_basics/

To implement:
- [ ] Text searchable
- [ ] Screen readable
- [ ] Collapsible branches for improved readability
- [ ] Alternate color modes (including black and white) for those with limited color perception
- [ ] A guided audio walk-through
- [ ] Reduced visual noise by creating more space in-between each item
- [ ] Increased context for each map item (is it a title, is it an example, is it a dependency, etc.)

What other accessibility improvements could be made?
